modules/installation-minimum-resource-requirements.adoc

```diff
@@ -245,20 +245,17 @@ ifdef::vsphere[]
 2. As with all user-provisioned installations, if you choose to use {op-system-base} compute machines in your cluster, you take responsibility for all operating system life cycle management and maintenance, including performing system updates, applying patches, and completing all other required tasks. Use of {op-system-base} 7 compute machines is deprecated and has been removed in {product-title} 4.10 and later.
 endif::vsphere[]
 --
-[NOTE]
-====
-As of {product-title} version 4.13, RHCOS is based on RHEL version 9.2, which updates the micro-architecture requirements. The following list contains the minimum instruction set architectures (ISA) that each architecture requires:
+
+ifdef::azure[]
+[IMPORTANT]
```

Comment:
I am not sure what you are doing here and there doesn't seem to be a preview for Azure.

Reply:
removed :)